src/main/resources/db-master-changelog.xml

```diff
@@ -1585,4 +1585,21 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20241202 Add list of all contextIds a node has ever had" author="Gunnar Velle">
+        <addColumn tableName="node">
+            <column name="contextids" type="jsonb" defaultValue="[]"/>
+        </addColumn>
+        <sql>
+            UPDATE node n
+            SET contextids=(
+            SELECT json_agg(obj->>'contextId')
+            FROM jsonb_array_elements(contexts) AS obj
```

Comment:
Vet ikke om jeg forstår koden helt ennå så ikke skyt meg om dette er dumt 😄 

Dette betyr at vi henter `contextids` fra en annen kolonne på samme rad sant?

Hvorfor kan vi ikke bare bruke `node.getContexts().map(ctx -> ctx.getId())` eller laget en helper på node for å gjøre det? Vil vel ikke koste stort mer ytelse og da slipper vi å holde dataen i sync?

Reply:
Heile poenget er at vi må ha en liste med kontekstider som _ikkje_ lenger er i lista med kontekster. For at vi skal kunne hente noder basert på gamle kontekster, som den nye testmetoden viser.